js/pages/cohort-definitions/components/reporting/inclusion-reports/inclusion-reports.js

```diff
@@ -22,16 +24,18 @@ define([
 					title: 'By Person',
 					componentName: 'feasibility-report-viewer-with-header',
 					componentParams: {
-						sourceName: params.sourceName,
-						report: ko.computed(() => params.report() && params.report().byPerson)
+						source: ko.computed(() => params.source()),
+						cohortId: ko.computed(() => params.cohort().id()),
+						reportType: constants.INCLUSION_REPORT.BY_PERSON,
 					},
 				},
 				{
 					title: 'By Events',
 					componentName: 'feasibility-report-viewer-with-header',
 					componentParams: {
-						sourceName: params.sourceName,
-						report: ko.computed(() => params.report() && params.report().byEvent)
```

Comment:
This is a strange shift:  the component for viewing the report takes a report data structure to the component. It doesn't care about the source (aside from using it as a report title, which could have been put intot he report object) and it doesn't care about the cohort Id (also, could be some meta about the cohort int he report object such as the name).

The architecture of these 'render' components has always been to be detached from the mechanism that loads the data.  Ie: you wouldn't put a service call to some 'scatterplot loader function' inside of the scatterplot renderer:  you just pass in the data that you want to render.  These report-type components should work the same, with the 'owning' component being the one to decide how to pipe the data into the component.  Separation of concerns.